interop-tests/src/test/scala/akka/grpc/scaladsl/NonBalancingIntegrationSpec.scala

```diff
@@ -34,14 +34,14 @@ class NonBalancingIntegrationSpec(backend: String)
     with BeforeAndAfterAll
     with ScalaFutures {
   implicit val system = ActorSystem(
-    "NonBalancingIntegrationSpec",
+    s"NonBalancingIntegrationSpec-$backend",
     ConfigFactory.parseString(s"""akka.grpc.client."*".backend = "$backend" """).withFallback(ConfigFactory.load()))
   implicit val mat = SystemMaterializer(system).materializer
   implicit val ec = system.dispatcher
 
-  override implicit val patienceConfig: PatienceConfig = PatienceConfig(5.seconds, Span(10, org.scalatest.time.Millis))
+  override implicit val patienceConfig: PatienceConfig = PatienceConfig(3.seconds, Span(10, org.scalatest.time.Millis))
```

Comment:
```suggestion
  override implicit val patienceConfig: PatienceConfig = PatienceConfig(5.seconds, Span(10, org.scalatest.time.Millis))
```

Patience can't hurt: we never wait for the full 5 seconds for 'successful' tests, only when failing (and then it's fine)